src/actions/guides/database/validating-saving-deleting.cr

```diff
@@ -274,18 +274,25 @@ class Guides::Database::ValidatingSavingDeleting < GuideAction
     * `validate_uniqueness_of` - to only allow one record with a field's value
 
     > Note: non-nillable (required) fields automatically use
-    `validate_required`. They will run after the `prepare` callback. This way data
+    `validate_required`. They will run after all other `before_save` callbacks have run. This way data
     with missing fields will never be sent to the database.
 
     ### Using validations
 
-    You can use validations inside of the `prepare` callback:
+    You can use validations inside of `before_save` callbacks:
 
     ```crystal
-    class UserForm < User::BaseForm
-      fillable name, password, password_confirmation, terms_of_service, age
+    class SaveUser < User::SaveOperation
+      permit_columns name, password, password_confirmation, terms_of_service, age
+
+      before_save validate_data
```

Comment:
I think we should flip these examples and do a do/end for validation and a custom method for the other one like “reject_scary_monsters” or something. I think it’ll be fairly common to not name validation blocks since they’ll be so common